patches/node/chore_disable_improvestack_regex.patch

```diff
@@ -0,0 +1,32 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Shelley Vohr <shelley.vohr@gmail.com>
+Date: Wed, 1 Feb 2023 19:32:56 +0100
+Subject: chore: disable improveStack regex
+
+Causing crashes in ARM
+
+diff --git a/lib/internal/util/inspect.js b/lib/internal/util/inspect.js
+index 6174d62bb11643949bfb041db36b94f1998d0df9..ac01a55a86117d91b121f9679ee78e5dfda5bf68 100644
+--- a/lib/internal/util/inspect.js
++++ b/lib/internal/util/inspect.js
+@@ -1271,13 +1271,13 @@ function improveStack(stack, constructor, name, tag) {
+       StringPrototypeStartsWith(stack, name) &&
+       (stack.length === len || stack[len] === ':' || stack[len] === '\n'))) {
+     let fallback = 'Error';
+-    if (constructor === null) {
+-      const start = RegExpPrototypeExec(/^([A-Z][a-z_ A-Z0-9[\]()-]+)(?::|\n {4}at)/, stack) ||
+-      RegExpPrototypeExec(/^([a-z_A-Z0-9-]*Error)$/, stack);
+-      fallback = (start && start[1]) || '';
+-      len = fallback.length;
+-      fallback = fallback || 'Error';
+-    }
++    // if (constructor === null) {
++    //   const start = RegExpPrototypeExec(/^([A-Z][a-z_ A-Z0-9[\]()-]+)(?::|\n {4}at)/, stack) ||
```

Comment:
Nice work 👏 

Does this trigger a crash in d8 ? What are the next steps for an upstream issue ?

Reply:
ah it's not this change - i pushed this for testing! i did actually find the offending regex last night though so i'll push that up this morning to confirm. Next steps are to reproduce it in d8 and see if i can find an alternative regex to accomplish the same goal as the one causing the crash